src/ui/components/menu/SettingsMenuButton.tsx

```diff
@@ -24,6 +24,12 @@ export interface SettingsMenuButtonProps {
 export const SettingsMenuButton = (props: React.PropsWithChildren<SettingsMenuButtonProps>) => {
   const { children, menuStyle, icon } = props;
   const context = useContext(PlayerContext);
+
+  // Do not render an empty menu
+  if (React.Children.toArray(children).length === 0) {
```

Comment:
Use [`count()`](https://react.dev/reference/react/Children#children-count) instead.

```suggestion
  if (React.Children.count(children) === 0) {
```